Jenkinsfile_nightly

```diff
@@ -44,13 +44,11 @@ withNightlyPipeline("nodejs", product, component) {
     after('build') {
         sh 'mkdir -p functional-output'
         sh 'yarn setup'
-        sh 'yarn test-accessibility'
```

Comment:
Is this intentionally removed - we are still running the accessibility tests somewhere are we?

Reply:
yes

Reply:
not required in nightly pipeline